src/btf.h

```diff
@@ -29,6 +30,15 @@ struct btf_type;
 
 namespace bpftrace {
 
+// Note: there are several prefixes for raw tracepoint BTF functions.
+// "__probestub_" seems to be the most accurate in terms of getting the params
+// but it wasn't added until May 2023 so older kernels might not have it,
+// which is why we also check "__traceiter_" (as needed).
```

Comment:
Could we add links to some kernel docs here? Or at least the patch sets that added the `__probestub_` and `__traceiter_` prefixes?

Reply:
Yeah, I'll try to find something to add as a reference.